docs/api/browser-window.md

```diff
@@ -169,6 +169,14 @@ Emitted when window is minimized.
 
 Emitted when window is restored from minimized state.
 
+### Event: 'resize'
+
+Emitted when window is resized.
```

Comment:
The `resize` event will be triggered during the process across all platforms. But for `move` on OSX it will be triggered once it is moved, whereas it is triggered during on linux/windows . Need a better hook for move on those two platforms for consistent behaviour. Will add these to the docs, thanks!